src/test/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticleTest.java

```diff
@@ -38,13 +41,107 @@ public void tearDown(TestContext testContext) {
   }
 
   @Test
-  public void testGetQuotes(TestContext testContext) {
-    webClient.get("/quotes")
-        .as(BodyCodec.jsonArray())
-        .send(testContext.asyncAssertSuccess(response -> {
-          testContext.assertEquals(200, response.statusCode(), response.bodyAsString());
-          JsonArray quotes = response.body();
-          testContext.assertFalse(quotes.isEmpty());
-        }));
+  public void testGetQuotes(TestContext context) {
+    final Async async = context.async();
+    vertx.createHttpClient().getNow(PORT, HOST_NAME, QUOTES_ROUTE, response -> {
```

Comment:
No specific reason actually, I had used the HttpClient in a prior project, so used it directly over here. From what I understand, WebClient extends HttpClient and has methods which is helpful with web right? 
Shall I change it to use the WebClient implementation?

Reply:
The `WebClient` has been created because the `HttpClient`, while very flexible, was not very user-friendly. For example, proper error management is much easier with the `WebClient`.

Reply:
No, don't reimplement with the `WebClient`, I was just curious why you changed it.